Uyanda.Coffee.Application/Features/BeverageManagement/Persistence/IBeverageAccessor.cs

```diff
@@ -11,24 +11,20 @@ public interface IBeverageAccessor
 
         Task<BeverageSizeCostModel[]> GetBeverageCostAsync();
 
-        //Task<InvoiceModel> PurchaseAsync(IEnumerable<LineItemModel> lineItems, CustomerModel customer, bool IsRedeemingPoints);
-
         Task<CustomerModel> AddCustomerAsync(CustomerModel customer);
 
         Task<CustomerModel> GetCustomerAsync(CustomerModel customer);
 
         Task<CustomerModel> GetCustomerIdAsync(CustomerModel customer);
 
-        Task<IDictionary<int, decimal>> BeveragePricesAync();
-
-        Task<bool> DoesCustomerExistAsync(CustomerModel customer);
+        Task<IDictionary<int, decimal>> BeveragePricesAsync();
 
         Task<InvoiceModel> SimplePurchaseAsync(int customerId, IEnumerable<LineItemModel> lineItems,  decimal cost, decimal exchangeRate, string Currency);
 
         Task<InvoiceModel> DiscountPurchaseAsync(int customerId, IEnumerable<LineItemModel> lineItems, decimal points, decimal cost, decimal exchangeRate, string Currency);
 
         Task UpdateCustomerPointsAsync(int customerId, decimal points);
 
-        Task<decimal> PayAsync(PayModel pay);
+        Task<decimal> PayAsync(PaymentModel pay);
```

Comment:
I would expect the paramenter name to now be _payment_

Reply:
Renamed to _PaymentAsync(PaymentModel payment)_